troubleshooting/nuget-feed.md

```diff
@@ -60,7 +62,23 @@ If you suspect that your saved credentials are wrong, use the following steps to
 
 ## Error 401 Unauthorized
 
-If your credentials are correct and your license includes the requested product and version, then the password probably contains special characters. You need to escape these characters or the authentication can fail on the NuGet server. For example, a common character you must escape is the ampersand (`&`); however, the character causing the issue may be as unique as the section character (`§`).
+`Error 401 Unauthorized` means that the Telerik NuGet server received invalid credentials. There may be different reasons for that:
+
+* No provided credentials
+* Incorrect password
+* [Correct password with unescaped special characters](#special-characters-in-the-password)
+* Using an invalidated (removed) [Telerik NuGet API key]({%slug installation/nuget%}#use-nuget-api-key), which no longer exists in <a href="https://www.telerik.com/account/downloads/nuget-keys" target="_blank">your Telerik account</a>.
+* Using a valid Telerik NuGet API key with the wrong username. It must be `api-key`.
+
+An easy way to verify your credentials is to [access the Telerik NuGet server directly in the web browser](#tips-for-handling-common-nuget-issues). Then, depending on your setup, check or update your credentials in:
```

Comment:
It would be more helpful and logical for the link to point to `#verify-nuget-credentials-and-package-access` or to the actual NuGet server.